proto/executionapis/astria/execution/v1alpha2/execution.proto

```diff
@@ -86,6 +84,8 @@ message CommitmentState {
   Block soft = 1;
   // Firm commitment is achieved when data has been seen in DA.
   Block firm = 2;
+  // The lowest block number of celestia chain to be searched for rollup blocks given current state
+  uint64 base_celestia_height = 3;
```

Comment:
is this equal to the celestia height the firm block was in? it seems so from the code but want to double check

Reply:
Right now yes, long term not necessarily. This is ok right now because sequencer relayer posts such that astria blocks are in order in celestia blocks and we have a single relayer. Long term need to add more complex logic into conductor with timestamps and acceptable time smear.